src/app-sync.js

```diff
@@ -13,8 +13,13 @@ import { SyncProtoBuf } from '@actual-app/crdt';
 
 const app = express();
 app.use(errorMiddleware);
+app.use(express.json());
+
```

Comment:
The `/delete-user-file` `req.body` was not being parsed as `json` by default, which necessitates the middleware to be added.

I manually tested that the `/sync` endpoint still works by launching the frontend add doing some syncable stuff, and watching the requests/responses.
It would be nice if there were tests for this function, which is my next plan.